app/components/EmissionInfoButton/EmissionInfoButton.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { useSelector } from "react-redux";
+import { pathOr } from "ramda";
+import { Ionicons } from "@expo/vector-icons";
+import { useNavigation, useRoute } from "@react-navigation/native";
+
+import { emissions } from "../../ducks";
+import { navigate } from "../../navigation";
+import styles from "./EmissionInfoButton.styles";
+
+const EmissionInfoButton = () => {
+  const navigation = useNavigation();
+  const navigator = navigate(navigation);
+
+  const route = useRoute();
+  const emissionId = pathOr("", ["params", "id"], route);
+
+  const { emissionType } =
```

Comment:
const { emissionModelType } is what we want (I think you haven't fully understand the task)

Reply:
You're correct, I didn't realize that based on the issue. Switching to that